src/main/kotlin/com/sourcegraph/cody/edit/FixupService.kt

```diff
@@ -45,13 +75,20 @@ class FixupService(val project: Project) : Disposable {
     TestCodeSession(this, editor, editor.project ?: return)
   }
 
+  @RequiresEdt
```

Comment:
Until the annotation processor supports Kotlin, should we back these up with manual assertions?